src/pretalx/static/common/scss/_forms.scss

```diff
@@ -276,7 +274,6 @@ table .action-column {
     border: 1px solid #ced4da;
     border-top: none;
     min-height: 160px;
-    max-height: 160px;
```

Comment:
Note that with this line change, the text area on preview is expanded to the size of the containing markdown, which may not be preferred as it's not a backward compatible change.

The other two above are fine though.